ynr/apps/bulk_adding/static/bulk_adding/css/bulk.scss

```diff
@@ -70,9 +70,6 @@
 .sopn_adding_suggestions_table {
     td {
         vertical-align: top;
-        &:first-child {
-            width: 50%;
-        }
```

Comment:
This is causing the names in the first column to get squished up 
![image](https://user-images.githubusercontent.com/10633357/141514230-aa3ca63b-4a06-4dd4-ab55-4a038bd721cc.png)

Reply:
I think adding `style="overflow-x:auto;"` to the `sopn_adding_suggestions_table` will get you the responsiveness and width you're after for the name column (rather than hardcoding the width).